### DISCUSSION TITLE
Why babel remove lerna？

### DISCUSSION THREAD

**Discussion by Sandra310** [babel/babel#12622] [Q&A] [answered]
I'm investigating whether I can manage the component library through lerna. I want to refer to babel and found that babel no longer uses lerna. I want to know if there are any scenes or why I consider giving up, I want to be a reference

**Comment by fedeci** [comment]
Hi, you can always check git history to see how lerna was configured on Babel!
Right now they are using [yarn-plugin-babel-release-tool](https://github.com/nicolo-ribaudo/yarn-plugin-babel-release-tool/) along with GitHub actions.

**Comment by nicolo-ribaudo** [comment, marked as answer]
1. we have been running on a patched lerna version for years, because lerna doesn't support anymore publishing only changed packages while keeping the versions in sync
2. Yarn workspaces are way faster than `lerna bootstrap` in this repository
3. We needed to use Yarn's `workspace:` protocol, but lerna doesn't support it
4. We were using a very small subset of Lerna's features
